signer/src/schnorr.rs

```diff
@@ -107,10 +111,10 @@ impl<H: 'static + Hashable> Schnorr<H> {
         let mut blake_hasher = Blake2bVar::new(32).unwrap();
 
         let mut roi: ROInput = input.to_roinput();
-        roi.append_field(kp.public.into_point().x);
```

Comment:
`kp.public` is implicitly copied twice before, now only `x` and `y` are implicitly once respectively